kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryValueProcessorImpl.java

```diff
@@ -23,6 +23,8 @@
     @Autowired
     private MctsChildDataService mctsChildDataService;
 
+    private final int msisdnLength = 10;
```

Comment:
Probably a good idea to move this constant declaration to the utils/KilkariConstants class